src/style.css

```diff
@@ -1 +1,74 @@
 /* add styles here */
+
+body {
+  margin: 0;
+  padding: 0;
+
+  /* padding: 22px; */
+  flex-direction: column;
+  align-items: center;
+
+  /* background-color: gray; */
+
+  /* display: flex; */
+
+  /* justify-content: center; */
+
+  /* align-items: center; */
+
+  /* height: 100vh;  */
+}
+
+.stars {
+  display: flex;
+
+  /* align-items: center;
+  justify-content: center;
+  text-align: center; */
+}
+
+.stars__star {
+  /* #параметри з фігма */
+  width: 16px;
+  height: 16px;
+  top: 42px;
+  left: 22px;
+
+  /* object-position: top left; */
+
+  /* margin: 2px;
+  margin-top: 0; */
+
+  margin-right: 4px;
+
+  /* margin-bottom: 2px; */
+
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+
+  background-position: center center;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The nth-child selector `:nth-child(-n + 0)` will not select any elements. If this is intentional to represent zero active stars, it is correct. Otherwise, you might want to adjust it.